mmv1/products/compute/terraform.yaml

```diff
@@ -335,6 +335,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
   RegionBackendService: !ruby/object:Overrides::Terraform::ResourceOverride
     examples:
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Can we move this Example towards the bottom of the examples block? The order listed here is how they will appear in the documentation, and we should generally have the basic version of the resource listed first

Reply:
Done